### DISCUSSION TITLE
tuple index out of range error(python 3.10, pyinstaller 4.7)

### DISCUSSION THREAD

**Discussion by Kingchobab** [pyinstaller/pyinstaller#6370] [Help] [answered]
I can't execute my python file.

D:\Github>pyinstaller --onefile --noconsole EggFinder_Beta.py
**62 INFO: PyInstaller: 4.7
62 INFO: Python: 3.10.0**
_...(syncopation)..._
4566 INFO: site: retargeting to fake-dir 'C:\\Users\\Kimms\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\PyInstaller\\fake-modules'
Traceback (most recent call last):
  File "C:\Users\Kimms\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
_...(syncopation)..._
  File "C:\Users\Kimms\AppData\Local\Programs\Python\Python310\lib\dis.py", line 292, in _get_const_info
    argval = const_list[const_index]
**IndexError: tuple index out of range**

I looked for people who had the same problem as me on Google, but I couldn't solve this problem with the way they solved it.
What I've checked so far is not a version of the problem. Other files were able to generate exe files normally.
Only "EggFinder_Beta.py" makes error.
I don't know how to upload my source code to GitHub.
I used py2exe because I failed to execute by pyinstaller, and a similar error message appeared in this case.
But when I removed the file included in the code, there was no error. However, there are some problems with the files created because I am not fully familiar with py2exe(It's really a small problem. I definitely designated the option as "one file," but the result was VERY messy.).
So, How can I solve this problem?

(I'm sorry if there's anything I don't understand. I am poor at English. And I barely know how to use Github.)

**Comment by rokm** [comment, marked as answer]
That's python 3.10.0 bug; see #6301. Either wait for python 3.10.1 to be released, or manually patch your python's `dis` module as described in the linked issue.

**Reply by Kingchobab** [reply]
Other files are normally be the latest version of it this problem is that I don't know... I have conducted from 3.7 (Anaconda virtual environment) version, and I made it. Thank you.
